clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/TitusUtils.java

```diff
@@ -51,4 +51,14 @@ public static void assertTitusAccountCredentialsType(AccountCredentials accountC
               accountCredentials.getName()));
     }
   }
+
+  /** Get registry details for a particular Titus account. */
+  @Nonnull
+  public static String getRegistry(
+      @Nonnull AccountCredentialsProvider accountCredentialsProvider, @Nonnull String credentials) {
+    AccountCredentials accountCredentials = accountCredentialsProvider.getCredentials(credentials);
+    // Assert that AccountCredentials of NetflixTitusCredentials type
```

Comment:
nit: I don't see value in code comments like this, since the code itself is documenting the same thing.

Reply:
fixed this